pkg/controllers/sidecar/config/config.go

```diff
@@ -4,8 +4,10 @@ import (
 	"context"
 	"fmt"
 	"io/ioutil"
+	pointer2 "k8s.io/utils/pointer"
```

Comment:
pointer2? package is called `pointer`, i don't see name conflict in mports

Reply:
fixed